src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IParenthesized.cs

```diff
@@ -496,5 +496,258 @@ static int M1()
 
             VerifyOperationTreeAndDiagnosticsForTest<ReturnStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
```

Comment:
There are no IParenthesizedOperations in any of these tests, so they don't tell us anything about that operation. Please update the tests to have scenarios that actually generate IParenthesizedOperation nodes.

Reply:
> There are no IParenthesizedOperations in any of these tests

At the moment C# erases them completely. I explicitly asked to add these tests anyway. At some point we were trying to synthesize the nodes for C# as well, but that became to risky and expensive at the time. We might still have issue to revisit that. 

---
In reply to: [187684261](https://github.com/dotnet/roslyn/pull/26801#discussion_r187684261) [](ancestors = 187684261)